documentation.html

```diff
@@ -0,0 +1,1440 @@
+          <h2 class="mt-2">(4) Flex</h2>
+          <table border="1" class="mt-1">
+            <thead>
+              <th>Property</th>
+              <th>Utility</th>
+            </thead>
+            <tbody>
+              <tr>
+                <td>display: flex;</td>
+                <td>flex</td>
+              </tr>
+              <tr>
+                <td>justify-content: space-between</td>
+                <td>justify-sb</td>
+              </tr>
+              <tr>
+                <td>align-items: center</td>
+                <td>align-center</td>
+              </tr>
+              <tr>
+                <td>flex-direction: column</td>
+                <td>flex-column</td>
+              </tr>
+              <tr>
+                <td>flex-wrap: wrap;</td>
+                <td>flex-wrap</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <div id="text">
+          <h1 class="uppercase letter-spaced mt-2">Text Utilities</h1>
+          <div class="mt-1">
+            <h1 class="font-size-xl extra-bold mt-02">
+              This is extra large text with extra bold weight
+            </h1>
+            <h1 class="font-size-l bold mt-02">
+              This is large text with bold weight.
+            </h1>
+            <h1 class="font-size-m medium mt-02">
+              This is medium text with medium weight.
+            </h1>
+            <h1 class="font-size-sm light mt-02">
+              This is small text with light weight.
+            </h1>
+            <h1 class="font-size-xsm extra-light mt-02">
+              This is extra small text with extra light weight.
+            </h1>
+            <p class="font-size-m medium grayText mt-02">This is gray text.</p>
+            <p class="font-size-m medium center mt-02">
+              This is centered text.
+            </p>
+            <p class="font-size-m medium uppercase mt-02">
+              this is uppercase text.
+            </p>
+            <p class="font-size-m medium letter-spaced mt-02">
+              This is text with some letter spacing.
+            </p>
+          </div>
+          <pre class="m-1-0"><code class="html border-round">
+&lt;h1 class="font-size-xl extra-bold"&gt;This is extra large text with extra bold weight&lt;/h1&gt;
+&lt;h1 class="font-size-l bold"&gt;This is large text with bold weight.&lt;/h1&gt;
+&lt;h1 class="font-size-m medium"&gt;This is medium text with medium weight.&lt;/h1&gt;
+&lt;h1 class="font-size-sm light"&gt;This is small text with light weight.&lt;/h1&gt;
+&lt;h1 class="font-size-xsm extra-light"&gt;This is extra small text with extra light weight.&lt;/h1&gt;
+&lt;p class="font-size-m medium grayText"&gt;This is gray text.&lt;/p&gt;
+&lt;p class="font-size-m medium center"&gt;This is centered text.&lt;/p&gt;
+&lt;p class="font-size-m medium uppercase"&gt;this is uppercase text.&lt;/p&gt;
+&lt;p class="font-size-m medium letter-spaced"&gt;This is text with some letter spacing.&lt;/p&gt;
```

Comment:
the more readable way would be using js to render text html , this will help you whenever you need to update the component's style or markup ( you can directly look into js file for what to change, obviously we can do it here but it takes more time)

new approach
in index.html
```
<code class="typography-classes-html">
</code>
```

in js file
```
let typography_markup = document.getElementByClassName('typography-classes-html')[0]

document.addEventListener('DOMContentLoaded',()=> {
   // you can even use for loop to iterate through all typography
   typography_markup.textContent = `<h1 class="font-size-xl extra-bold">This is extra large text with extra bold weight</h1>`
})
```

benefits
1. impoved readbiltity
2. saves time
3. much more optimal way (follows DRY)